debian/control

```diff
@@ -38,6 +38,21 @@ Depends: ${misc:Depends}, ${shlibs:Depends}
 Description: This is securedrop Qubes proxy service
  This package provides the network proxy on Qubes to talk to the SecureDrop server.
 
+Package: securedrop-qubesdb
+Section: admin
+Architecture: all
+Depends: ${misc:Depends}, python3, python3-qubesdb
+Description: Tools for configuring non-Qubes-aware applications from QubesDB.
+ This package provides tools for configuring non-Qubes-aware applications from
+ QubesDB.
+
+Package: securedrop-whonix-config
+Section: admin
+Architecture: all
+Depends: ${misc:Depends}, securedrop-qubesdb, tor
```

Comment:
These `Depends` lines cause `piuparts` to fail [on `python3-qubesdb`](https://github.com/freedomofpress/securedrop-client/actions/runs/9217003898/job/25358352091#step:4:1960) and [on `securedrop-qubesdb`](https://github.com/freedomofpress/securedrop-client/actions/runs/9217003898/job/25358352091#step:4:1960), respectively.  @legoktm, do you have advice on how we should handle this, given that (a) we do want these dependencies enforced at installation but (b) they will indeed not be checkable by `piuparts` in CI?

Reply:
We should enable the Qubes repositories in piuparts - this probably means we're (technically) missing the correct dependencies in the proxy and client packages already - let me poke.

Reply:
Thanks!